docker/pulumi/Dockerfile

```diff
@@ -36,11 +39,17 @@ RUN apt-get update -y && \
 
 # Install cloud tools
 RUN \
+  # Setup environment variables for architecture-specific packages
+  if [ "$TARGETARCH" = "amd64" ]; then \
+    AWSCLI_ARCH=x86_64; \
+  else \
+    AWSCLI_ARCH=aarch64; \
```

Comment:
We should probably switch the default around here, so amd64 is the default?  Or maybe set that as a default above if no TARGETARCH argument is given?